src/section-adding-and-subtracting-polynomials.ptx

```diff
@@ -212,7 +211,7 @@
         </statement>
         <solution>
             <p>We must first distribute the <m>-1</m> through <m>\left(-3x^2+9x-2\right)</m> and then we can combine like terms.<md>
-                <mrow>\left(5x^3+4x^2-6x\right)-\left(-3x^2+9x-2\right) \amp= 5x^3+4x^2-6x\highlight{{}+3x^2-9x+2}</mrow>
+                <mrow>\left(5x^3+4x^2-6x\right)-\left(-3x^2+9x-2\right) \amp= 5x^3+4x^2-6x \highlight{+} 3x^2 \highlight{-} 9x \highlight{+} 2</mrow>
```

Comment:
Using `\highlight{}` kills the scope of things inside. So with `\highlight{+}`, that plus sign doesn't know there are things to its left and right, so it will not put adequate spacing to its left and right. These should be like `\highlight{{}+{}}`.